client/src/state/index.js

```diff
@@ -65,15 +65,32 @@ export type MapOptions = {
 
 export type MapReference = any
 
+export type EntitySearch = {
+  eids: string[],
+  id: string,
+}
+
+export type Connections = {
+  entities: {[string]: Entity},
+  detail: {[string]: {ids: string[]}},
+  entityDetails: {
+    [string]: {
+      name: string,
+      data: any, //TODO: TBD
```

Comment:
oh, I see now - ok, disregard the Annies if they are related to this

Reply:
Yup, they are :slightly_smiling_face: